src/graphql/root/mutation/onchain-usd-payment-send.ts

```diff
@@ -0,0 +1,86 @@
+import { GT } from "@graphql/index"
+import Memo from "@graphql/types/scalar/memo"
+import { mapAndParseErrorForGqlResponse } from "@graphql/error-map"
+import WalletId from "@graphql/types/scalar/wallet-id"
+import CentAmount from "@graphql/types/scalar/cent-amount"
+import OnChainAddress from "@graphql/types/scalar/on-chain-address"
+import PaymentSendPayload from "@graphql/types/payload/payment-send"
+import TargetConfirmations from "@graphql/types/scalar/target-confirmations"
+import { validateIsUsdWalletForMutation } from "@graphql/helpers"
+import { Wallets } from "@app"
+
+const OnChainUsdPaymentSendInput = GT.Input({
+  name: "OnChainUsdPaymentSendInput",
+  fields: () => ({
+    walletId: { type: GT.NonNull(WalletId) },
+    address: { type: GT.NonNull(OnChainAddress) },
+    amount: { type: GT.NonNull(CentAmount) },
+    memo: { type: Memo },
+    targetConfirmations: { type: TargetConfirmations, defaultValue: 1 },
+  }),
+})
+
+const OnChainUsdPaymentSendMutation = GT.Field<
+  {
+    input: {
+      walletId: WalletId | InputValidationError
+      address: OnChainAddress | InputValidationError
+      amount: number
+      memo: Memo | InputValidationError | null
```

Comment:
if nothing is passed, it default to `null`, not `undefined`?

Reply:
Not sure which is appropriate here. This was copied from the existing other 2 onchain-send mutations and it looks like we use `null` in a few other places in the graphql layer.